### ISSUE TITLE
Rebar3 lfe repl doesn't start apps in shell option in 0.4.6 and 0.4.7

### ISSUE THREAD

**Issue by lamuswawir** [lfe/rebar3#85]
`Rebar3 lfe repl` doesn't start applications listed under `apps` in the `shell` option of `rebar.config` in versions `0.4.6` and `0.4.7` although it works in `0.4.5`.

My setup:
OTP 25.3.2.5
LFE 2.1.2

**Comment by oubiwann** [comment]
How about 0.4.8? 

If it worked in 0.4.5, I wonder if it might be related to the changes / workaround we had to make for Erlang 26 ... (those workarounds affected interactions with Erlang 25, too). 

Hopefully this will become a non-issue once the related fixes made to rebar3 have been released (they are committed, but last I looked, rebar3 hadn't created a release with those changes yet). Not sure this is related to what you're seeing, but it it is, there's some more context here: #82
